sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/utils/CxxSqaleXmlLoader.java

```diff
@@ -0,0 +1,186 @@
+/*

```

Comment:
when i first look at that method i tought that thing could be used to read a file with the definitions. but no, so this is ok. my only concern is that we need a integration test to go with this and to cover also our own internal sqale model